src/main/java/uk/gov/hmcts/reform/bailcaseapi/domain/entities/BailCaseFieldDefinition.java

```diff
@@ -5,7 +5,9 @@
 public enum BailCaseFieldDefinition {
     //Example
     APPELLANT_GIVEN_NAMES(
```

Comment:
Do we need this enum in bail case api?

Reply:
Think it was just an example I'd forgotten to remove - thanks will update